all-repos.yaml

```diff
@@ -222,3 +222,4 @@
 - https://github.com/omnilib/ufmt
 - https://github.com/daveshanley/vacuum
 - https://github.com/nuztalgia/botstrap
+- https://gitlab.com/adam-moss/pre-commit-hooks
```

Comment:
shell limits the portability, especially for windows -- it would be preferable to implement these in one of the supported languages

also could you rename your repo to something more specific?  repos named `pre-commit-hooks` are easy to confuse with the framework and it'd be easier to find with a name specific to what you've written

Reply:
In terms of language I'm not look to re-implement these hooks at this time.  I know there may be issues under WSL but I don't have access to Windows based device to test efficiently or regularly, so any support would only every be on a reasonable endeavours basis.

Reply:
Renamed (and split) repo as requested.

Thanks